deploy/edge/zone.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: zone
+data:
+  named.conf: |
+    include "/etc/bind/ddns.key";
+    options {
+      directory "/var/cache/bind";
+      listen-on port 1053 { any; };
+      allow-query { any; };
+    };
+    zone "example.com" {
+      type master;
+      file "/var/lib/bind/zones/k8s.zone";
+      allow-transfer {
+          key "externaldns-key";
+      };
+      update-policy {
+          grant externaldns-key zonesub ANY;
+      };
+    };
+  k8s.zone: |
+    $TTL 30
+    @         IN SOA  example.com. root.example.com. (
+                                    16         ; serial
+                                    30         ; refresh (1 minute)
+                                    30         ; retry (1 minute)
+                                    30         ; expire (1 minute)
+                                    30         ; minimum (1 minute)
```

Comment:
nit: 30s  != 1 min

in the worst case, the TTL will be 30(bind)+30(core dns) = 60, right? 

those `cloud.example.com` records will be created by DNSendpoint CRs?

Reply:
Yes